ui/client/actions/esp/groups.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import {events} from "../../analytics/TrackingEvents"
+import type {ThunkAction} from "../types"
+import {reportEvent} from "./reportEvent"
+import type {GroupId, NodeId, NodeType} from "./types"
+
+export function startGrouping(): ThunkAction {
```

Comment:
This file can be merged with editGroup into one file that will contains group actions